AdminWebsite/AdminWebsite/Extensions/HearingDetailsResponseExtensions.cs

```diff
@@ -90,7 +90,13 @@ private static OtherInformationDetails GetOtherInformationObject(string otherInf
             }
             catch (Exception)
             {
-                return new OtherInformationDetails {OtherInformation = otherInformation};
+                var properties = otherInformation.Split("|");
+                if (properties.Length > 1)
+                {
+                    return new OtherInformationDetails {OtherInformation = properties[2]};
+                }
+
+                return new OtherInformationDetails {OtherInformation = properties[0]};
```

Comment:
same as = otherInformation, this might be clearer tho?

Reply:
Do you mean the parameter? That is a pipe separated string. To map the correct values to the object, we need to split the string into an array, then extract the corresponding values

Reply:
if it was pipe separated it would go inside the block above

Reply:
We can still get a pipe separated format when we have old data (ie no judge email/phone). The old data looks like this now `|OtherInformation|Sometext|`, this was picked up in testing. To cater for this, we need this if statement, otherwise the smoke test fails

Reply:
super, comment still stands tho, line 96 will set to [2], this one here is setting to `otherinformation` if it hasnt split as expected

Reply:
But if we do that the property within the object will equal `|OtherInformation|Sometext|`, we just want it to equal `Sometext`

Reply:
🙄 think we might be best to have a quick chat.. dont think were understanding each other here :)